tests/smoke_test.py

```diff
@@ -80,3 +80,35 @@ def emit(self, record):
     else:
         # Skip the test for 32-bit Linux systems
         print("Skipping test, 32-bit DDWAF not ready yet")
+
+    # Profiling smoke test
+    print("Running profiling smoke test...")
+    profiling_cmd = [sys.executable, "-c", "import ddtrace.profiling.auto"]
```

Comment:
is this enough? would a `time.sleep(60)` exercise more? or we really just want to be sure the native binaries load, and then we are good? (I assume everything will get loaded with auto, nothing lazy loaded on first flush?)

Reply:
I don't want to increase the test time by that much and just want to make sure we don't have any obvious crashes. Yeah, we don't do lazy loading.

Reply:
ok, not lazy loading then mostly seems good, since loading the native stuff is the hard part to validate, so 👍🏻